sync-s3-to-b2.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env sh
+
+MYNAME="sync-s3-to-b2"
+STATUS=0
+
+echo "${MYNAME}: Started"
+
+if [ "${B2_APPLICATION_KEY_ID}" = "" ]; then
+	echo "${MYNAME}: FATAL: environment variable B2_APPLICATION_KEY_ID is required."
+	STATUS=1
+fi
```

Comment:
Would a small function help reduce some of the duplication?

Reply:
And even an abstraction function to conglomerate the validations to a single line in the main code.

Reply:
I think I'll leave the abstraction function to a future developer.